input/fsh/Ex-XRTS-02-Prostate-2P-1V.fsh

```diff
@@ -19,7 +19,7 @@ Usage: #example
   * extension[technique][+].valueCodeableConcept = SCT#441799006 "Intensity modulated radiation therapy (procedure)"
 * extension[actualNumberOfSessions].valueUnsignedInt = 9
 * extension[doseDeliveredToVolume][+]
-  * extension[volume]
+  * extension[volume] 
```

Comment:
Looks like you are adding spaces at the end of lines?
I'm often guilty of this.   There is a VSCode setting to strip these, I think.